Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-labels bundle size difference from 5.0.2 to 0dcb2389015d2f9010c7a199d45a939f553a1138
No bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-labels bundle size difference from 5.0.2 to e0db58661171d081f6215e61a5bb9eb23e91f892
No bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
